eq-author/src/App/section/Design/SectionEditor/index.js

```diff
@@ -139,6 +142,11 @@ class SectionEditor extends React.Component {
               size="large"
               testSelector="txt-section-title"
               autoFocus={autoFocusTitle}
+              errorValidationMsg={this.props.getValidationError({
+                field: "title",
+                message:
+                  "Enter a section title. If section navigation is not required, disable in 'settings'.",
```

Comment:
Should it be `disable it in 'settings'.`?

Reply:
The text came from the card